src/frontend/src/catalog/system_catalog/rw_catalog/rw_table_stats.rs

```diff
@@ -22,9 +22,9 @@ use crate::error::Result;
 struct RwTableStats {
     #[primary_key]
     id: i32,
-    total_key_count: i64,
-    total_key_size: i64,
-    total_value_size: i64,
+    storage_key_count: i64,
```

Comment:
```
message TableStats {
  int64 total_key_size = 1;
  int64 total_value_size = 2;
  int64 total_key_count = 3;

  // `total_compressed_size`` represents the size that the table takes up in the output sst
  //  and this field is only filled and used by CN flushes, not compactor compaction
  uint64 total_compressed_size = 4;
}
```

Do you want to synchronously modify the naming in the proto? I have no preference either way.

Reply:
Field renaming is a break change in protobuf. To safely to that with backward compatibility, we need to introduce new fields with new id/name and deprecate the old ones, along with adding compatibility codes in the place where this pb is used. I would rather not touch that.